spec/requests/users_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'correct response status' do
+      get '/users'
+      expect(response).to be_successful
+    end
+
+    it 'check if correct template was rendered' do
+      get '/users'
+      expect(response).to render_template(:index)
+    end
```

Comment:
- [ ] Same applies to these ones too 
![image](https://github.com/musangamfure/Blog-app/assets/86200356/7ac5fa0c-9030-4d07-9e18-cc4211e19f28)